src/d3a/d3a_core/d3a_jobs.py

```diff
@@ -41,7 +41,7 @@ def decompress_and_decode_queued_strings(queued_string):
 
 @job('d3a')
 def start(scenario, settings, events, aggregator_device_mapping):
-    logging.getLogger().setLevel(logging.ERROR)
+    logging.getLogger().setLevel(logging.INFO)
```

Comment:
Hmm, I am not sure that we want to print info logs by default in production (we probably don't want to)

Reply:
Sorry, remnants of testing will set the log level back to ERROR